### DISCUSSION TITLE
Create Own DataSet

### DISCUSSION THREAD

**Discussion by garcipat** [bchavez/Bogus#430] [Q&A] [answered]
Hi

I just tried to create my own DataSet as in the Reame declared. But ContextHelper comes from Bogus.Premium. Is this a premium feature then to write my own DataSets?

Thanks for the clarification.

**Comment by bchavez** [comment, marked as answer]
Hey there. Thank you for the question. You can find more information at the link below -- the information in the GH issue is still accurate:
* https://github.com/bchavez/Bogus/issues/283

I hope that helps. If you have any more questions, feel free to let me know and I'll be happy to help.
